apps/web/vibes/soul/form/button-radio-group/index.tsx

```diff
@@ -21,12 +21,11 @@ export const ButtonRadioGroup = React.forwardRef<
 >(({ id, label, options, errors, className, ...rest }, ref) => {
   return (
     <div className={clsx('space-y-2', className)}>
-      {label !== undefined && label !== '' && <Label htmlFor={id}>{label}</Label>}
+      {label !== undefined && label !== '' && <Label>{label}</Label>}
```

Comment:
We should keep the `htmlFor` and use `useId`

Reply:
There's no input directly correlated to this label since it's for the group.

Reply:
If there is no input then we might not want to use a `<label>` and instead have a label for the group as I describe in this comment: https://github.com/makeswift/vibes/pull/370#discussion_r1882329415